.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Extract tag name
+      id: tag
+      run: echo "::set-output name=value::${GITHUB_REF##*/}"
+    - uses: actions/checkout@v2
+    
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1.1.2
+      with:
+        ruby-version: 2.6
+    
+    - name: Install dependencies
+      run: bundle install
+    
+    - name: Create package
+      run: bundle exec rake package
+
+    - name: Get Directory
+      id: folder
+      run: |
+        cd packaging/builds
+        echo ::set-output name=name::$(ls)
+    
+    - name: Get Packages
+      id: binaries
+      run: |
+        cd packaging/builds/${{ steps.folder.outputs.name }}
+        echo $PWD
+        find . -type f -name "*.deb"
```

Comment:
This seems to be a duplicate line that's not really doing anything?

Reply:
🤦‍♀️ sorry, this is also a debugging line